main/views.py

```diff
@@ -1,6 +1,14 @@
-from django.shortcuts import render
+from django.shortcuts import render, redirect
+from django.contrib.auth import logout
 
 
 # Create your views here.
 def homePage(request):
+    if request.user.is_authenticated:
+        return render(request, 'main/loggedInPage.html')
     return render(request, 'main/homePage.html')
+
+
+def logout_view(request):
+    logout(request)
+    return redirect('homePage')
```

Comment:
I'll add this comment here also:
https://github.com/redhat-beyond/HighTeach/pull/78#discussion_r1193092587